src/robotlibcore.py

```diff
@@ -137,11 +130,50 @@ def get_keyword_documentation(self, name):
             return inspect.getdoc(self.__init__) or ''
         kw = self.keywords[name]
         doc = inspect.getdoc(kw) or ''
-        if kw.robot_tags and not self._get_keyword_tags_supported:
+        if kw.robot_tags and not self.__get_keyword_tags_supported:
             tags = 'Tags: {}'.format(', '.join(kw.robot_tags))
             doc = '{}\n\n{}'.format(doc, tags) if doc else tags
         return doc
 
+    def get_keyword_types(self, keyword_name):
+        method = self.__get_keyword(keyword_name)
+        if method == {}:
```

Comment:
This is a pretty strange sentinel. Why not `None`?

Reply:
True, `None` would be better. Will change.